atoma-proxy/src/server/config.rs

```diff
@@ -14,6 +14,11 @@ pub struct AtomaServiceConfig {
     ///
     /// This field specifies the address and port on which the Atoma Proxy Server will bind.
     pub service_bind_address: String,
+
+    /// Password for the Atoma Proxy Service.
+    ///
+    /// This password is used to authenticate requests to the Atoma Proxy Service.
+    pub password: String,
```

Comment:
Hum, why do we need this ?

Reply:
We need some kind of autheticantion with the atoma-proxy. Since every single proxy will manage single client, it doesn't make sense to do some kind of JWT token auth. Simple password is enough. If we are worried that someone will read this config, we can store hash of the password and compare that.